Content.Server/ADT/Objectives/Systems/StealDnaConditionSystem.cs

```diff
@@ -66,8 +66,7 @@ private float GetProgress(EntityUid uid, MindComponent mind, StealDnaConditionCo
 
         // Подсчёт требуемых и имеющихся ДНК
         var count = ling.DNAStolen;
-        var result = (float)count / (float)comp.AbsorbDnaCount;
-        result = Math.Clamp(result, 0, 1);
-        return result;
+        count = Math.Clamp(count, 0, 1);
+        return count;
     }
```

Comment:
Можно же сделать так

```suggestion
var count = Math.Clamp(ling.DNAStolen, 0, 1);
    return count;
```

Reply:
Я не любитель переписывать чужой код